vanilla/vanilla
# Fix social connect button when translation strings are long

Related to vanilla/support#1863.

This PR fixes a problem where the 'social-connect' slider isn't visible when the translation string for "Connect" is too long. For instance, in French, the slider button is longer than the whole slider element. This adds a css fix to ensure that the slider button is never longer than the slider element.

### TO TEST
Try this test in both Bootstrap and Keystone.
1. Set your Vanilla to French and have at least one social connect addon enabled.
1. Navigate to `profile/connections`
1. Note that the "Connectez-vous" button obscures the slider element.
1. Check out this branch and verify that the button doesn't obscure the slider element.
